Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -16,4 +16,31 @@ extension String {
             $0.properties.isWhitespace
         })
     }
+
+    static func _tryFromUTF16(_ input: UnsafeBufferPointer<UInt16>) -> String? {
+        withUnsafeTemporaryAllocation(of: UInt8.self, capacity: input.count * 3) { contents in
+            var ptr = contents.baseAddress!
+            var count = 0
+            let error = transcode(input.makeIterator(), from: UTF16.self, to: UTF8.self, stoppingOnError: true) { codeUnit in
```

Comment:
Is transcode provided by the standard library?

Reply:
It is.